package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
-    "arsenal": "github:scality/Arsenal#c848d1f",
+    "arsenal": "github:scality/Arsenal#2d30243",
```

Comment:
I updated the arsenal version hash to match the latest one based on here https://github.com/scality/Arsenal/pull/970 after the recent changes, is this the right way to do it?